GEOSmetforce_GridComp/LDAS_Forcing.F90

```diff
@@ -5405,8 +5460,11 @@ subroutine parse_G5DAS_met_tag( met_path_in, met_tag_in, date_time, &
           
           use_Predictor = .true.
 
-       else
-          
+       elseif (tmp_tag(ii)(1:3)=='bkg') then
+
+          use_bkg = .true. 
+
```

Comment:
I have to think more about this, but can't we simply replace "use_Predictor" with "use_bkg"?  The purpose is the same... use lfo output from the Predictor segment of the GCM.
There would probably be issues with backward compatibility if we eliminated "use_Predictor" and the `Nx+-` syntax, but we may not need backward compatibility in this case.  
This requires more thought

Reply:
I feel that replacing use_Predictor seems to be more disruptive to the current code.